kancolle_auto.sikuli/kancolle_auto.py

```diff
@@ -367,16 +371,7 @@ def init():
         combat_item = combat_module.Combat(global_regions['game'], settings)
         default_quest_mode = 'sortie'
         log_success("Combat module started (Sortie mode)")
-    if settings['pvp_enabled'] and settings['combat_enabled']:
-        if settings['pvp_fleetcomp'] == 0 or settings['combat_fleetcomp'] == 0:
```

Comment:
This actually reminds me... The current implementation will cause issues if the user for some reason enters a `0` and any other valid value. I still see the utility in allowing `0` to disable the fleetcomp switcher, as well as checking to see if the set fleet for PvP and combat are equal to disable, so please re-implement this.